threejs/three-FirstPersonControls.d.ts

```diff
@@ -11,21 +11,21 @@ declare namespace THREE {
         object: THREE.Object3D;
         target: THREE.Vector3;
         domElement: HTMLCanvasElement;
+        enabled: boolean;
         movementSpeed: number;
         lookSpeed: number;
         noFly: boolean;
         lookVertical: boolean;
         autoForward: boolean;
         activeLook: boolean;
         heightSpeed: boolean;
-        heightCoef: boolean;
-        heightMin: boolean;
+        heightCoef: number;
+        heightMin: number;
+        heightMax: number;
         constrainVertical: boolean;
         verticalMin: number;
         verticalMax: number;
         autoSpeedFactor: number;
-        mouseX: number;
```

Comment:
i see these in a public sample: https://github.com/mrdoob/three.js/blob/cf584a60bdfd24c42eaa81d484533364742bda44/examples/js/controls/FirstPersonControls.js#L35